contracts-periphery/test/UmbraBatchSend.t.sol

```diff
@@ -89,19 +89,16 @@ abstract contract UmbraBatchSendTest is DeployUmbraTest {
     uint256 totalAmount = uint256(amount) + amount2;
     uint256 totalAmount2 = uint256(amount3) + amount4;
 
-    sendData.push(UmbraBatchSend.SendData(alice, address(token), amount, pkx, ciphertext));
-    sendData.push(UmbraBatchSend.SendData(bob, address(token), amount2, pkx, ciphertext));
     sendData.push(UmbraBatchSend.SendData(alice, address(token2), amount3, pkx, ciphertext));
     sendData.push(UmbraBatchSend.SendData(bob, address(token2), amount4, pkx, ciphertext));
+    sendData.push(UmbraBatchSend.SendData(alice, address(token), amount, pkx, ciphertext));
+    sendData.push(UmbraBatchSend.SendData(bob, address(token), amount2, pkx, ciphertext));
```

Comment:
Just curious why was this order changed?

Reply:
I was getting the `NotSorted` error. Switching the order was an easy way to sort it. Which do makes me wonder if the tests were not running properly before yet still passing?

Reply:
If the mock tokens are tokens that we deploy (as opposed to a fork test with constant token addrs), then more likely is that the nonce changed due to the deploy txs that now happen first, and as a result the token addresses changed and were no longer sorted

Reply:
oh cool, that makes sense. Thank you!